extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -463,6 +466,13 @@ export class AmpA4A extends AMP.BaseElement {
           if (!fetchResponse || !fetchResponse.arrayBuffer) {
             return null;
           }
+          // If the response has response code 204, collapse it.
+          if (fetchResponse.status == 200) {
```

Comment:
Mm.  As I look at the
```javascript
if (!fetchResponse || !fetchResponse.arrayBuffer) {
  return null;
}
```
block, I seem to recall that it is _also_ intended to be a no-content/collapse behavior handler.  Though I'm not sure how it can be triggered, because I _think_ that `fetchResponse` itself should be non-null, as should its `arrayBuffer`, any time we get a non-erroneous network response.  As I read through the promise chain handler hereafter, I'm also not convinced that we correctly deal with even this null case return value.  :-/

Anyway, it might be worth rolling those cases into this handler as well.  (But please be sure that the existing tests continue to pass, of course, because I'm not 100% certain on that analysis.)